windows/windows_update_install/windows_update_install.yml

```diff
@@ -16,6 +16,15 @@
           vars:
             create_current_test_folder: true
 
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: >-
+              Skip test case due to windows_update_msu_path is not provided 
```

Comment:
what if `windows_update_msu_path` exists but `windows_msu_share_point` not

Reply:
Thanks. Will add a checking parameter task and fail the case for this situation.

Reply:
Will also skip the test case if any of them does not exists.